ts/src/modetrade.ts

```diff
@@ -0,0 +1,2874 @@
+                            'client/statistics': 60,
+                            'client/info': 60,
+                            'client/statistics/daily': 60,
+                            'positions': 3.33,
+                            'position/{symbol}': 3.33,
+                            'funding_fee/history': 30,
+                            'notification/inbox/notifications': 60,
+                            'notification/inbox/unread': 60,
+                            'volume/broker/daily': 60,
+                            'broker/fee_rate/default': 10,
+                            'broker/user_info': 10,
+                            'orderbook/{symbol}': 1,
+                            'kline': 1,
+                        },
+                        'post': {
+                            'orderly_key': 1,
+                            'client/set_orderly_key_ip_restriction': 6,
+                            'client/reset_orderly_key_ip_restriction': 6,
+                            'order': 1,
+                            'batch-order': 10,
+                            'algo/order': 1,
+                            'liquidation': 1,
+                            'claim_insurance_fund': 1,
+                            'withdraw_request': 1,
+                            'settle_pnl': 1,
+                            'notification/inbox/mark_read': 60,
+                            'notification/inbox/mark_read_all': 60,
+                            'client/leverage': 120,
+                            'client/maintenance_config': 60,
+                            'delegate_signer': 10,
+                            'delegate_orderly_key': 10,
+                            'delegate_settle_pnl': 10,
+                            'delegate_withdraw_request': 10,
+                            'broker/fee_rate/set': 10,
+                            'broker/fee_rate/set_default': 10,
+                            'broker/fee_rate/default': 10,
+                            'referral/create': 10,
+                            'referral/update': 10,
+                            'referral/bind': 10,
+                            'referral/edit_split': 10,
+                        },
+                        'put': {
+                            'order': 1,
+                            'algo/order': 1,
+                        },
+                        'delete': {
+                            'order': 1,
+                            'algo/order': 1,
+                            'client/order': 1,
+                            'algo/client/order': 1,
+                            'algo/orders': 1,
+                            'orders': 1,
+                            'batch-order': 1,
+                            'client/batch-order': 1,
+                        },
+                    },
+                },
+            },
+            'requiredCredentials': {
+                'apiKey': true,
+                'secret': true,
+                'accountId': true,
```

Comment:
we would like to use the ccxt mode trade connector with freqtrade env var config: https://www.freqtrade.io/en/stable/configuration/#environment-variables